### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by slenguyen**
This PR adds documentation to the README about BBS migrations now that it is in public beta. 

- [ ] Did you write/update appropriate tests
- [ ] Release notes updated (if appropriate)
- [ ] Appropriate logging output
- [ ] Issue linked
- [ ] Docs updated (or issue created)
- [ ] New package licenses are added to `ThirdPartyNotices.txt` (if applicable)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,6 +61,30 @@ Refer to the [official documentation](https://docs.github.com/en/migrations/usin
 
 Refer to the [official documentation](https://docs.github.com/en/migrations/using-github-enterprise-importer/migrating-repositories-with-github-enterprise-importer/migrating-repositories-from-azure-devops-to-github-enterprise-cloud) for more details.
 
+### BitBucket Server and Data Center to GitHub Usage
```
Comment (timrogers) [inline README.md:64]:
```suggestion
### Bitbucket Server and Data Center to GitHub Usage
```

File: README.md [hunk 2]
```diff
@@ -55,6 +61,30 @@ Refer to the [official documentation](https://docs.github.com/en/migrations/usin
...
+1. Create Personal Access Token for the target GitHub org (for more details on scopes needed refer to our [official documentation](https://docs.github.com/en/migrations/using-github-enterprise-importer/preparing-to-migrate-with-github-enterprise-importer/managing-access-for-github-enterprise-importer)).
+
+2. Set the `GH_PAT`, `BBS_USERNAME`, and `BBS_PASSWORD` environment variables. (If your BitBucket Server instance runs on windows, set `SMB_PASSWORD`.)
```
Comment (timrogers) [inline README.md:67]:
```suggestion
2. Set the `GH_PAT`, `BBS_USERNAME`, and `BBS_PASSWORD` environment variables.

3. If your Bitbucket Server or Data Center instance runs on Windows, set the `SMB_PASSWORD` environment variable.
```

File: README.md [hunk 3]
```diff
@@ -55,6 +61,30 @@ Refer to the [official documentation](https://docs.github.com/en/migrations/usin
...
+
+3. Run the `generate-script` command to generate a migration script.
```
Comment (timrogers) [inline README.md:69]:
```suggestion
4. Run the `generate-script` command to generate a migration script.
```

File: README.md [hunk 4]
```diff
@@ -55,6 +61,30 @@ Refer to the [official documentation](https://docs.github.com/en/migrations/usin
...
+  --output FILENAME \
+  # Use the following options if your Bitbucket Server instance runs on Linux
+  --ssh-user SSH-USER --ssh-private-key PATH-TO-KEY
+  # Use the following options if your Bitbucket Server instance runs on Windows
+  --smb-user SMB-USER
+  # Use the following option if you are running a Bitbucket Data Center cluster or your Bitbucket Server is behind a load balancer
+  --archive-download-host ARCHIVE-DOWNLOAD-HOST
+```
+
+4. The previous command will have created a `migrate.ps1` PowerShell script. Review the steps in the generated script and tweak if necessary.
```
Comment (timrogers) [inline README.md:82]:
```suggestion
5. The previous command will have created a `migrate.ps1` PowerShell script. Review the steps in the generated script and tweak if necessary.
```

File: README.md [hunk 5]
```diff
@@ -55,6 +61,30 @@ Refer to the [official documentation](https://docs.github.com/en/migrations/usin
...
+
+5. The `migrate.ps1` script requires PowerShell to run. If not already installed see the [install instructions](https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell?view=powershell-7.2) to install PowerShell on Windows, Linux, or Mac. Then run the script.
```
Comment (timrogers) [inline README.md:84]:
```suggestion
6. The `migrate.ps1` script requires PowerShell to run. If not already installed see the [install instructions](https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell?view=powershell-7.2) to install PowerShell on Windows, Linux, or Mac. Then run the script.
```

**Review by timrogers — APPROVED** [review]
This looks great! ❤️ Thanks for spotting the gap and addressing it so quickly. I'm just going to slightly change the steps to make them easier to read.

**Review by dpmex4527 — APPROVED** [review]
LGTM!

**Pull request merged**